packages/es-components/src/components/containers/tabPanels/TabPanel.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { noop } from 'lodash';
+import { noop, findIndex } from 'lodash';
```

Comment:
Note to self for when we adapt these changes in the `next` branch we can use `Array.findIndex` instead of `lodash.findIndex`.

#229